index.rst

```diff
@@ -2,24 +2,54 @@
+We experienced a catastrophic network failure that took down the network in the entire observatory.
+The failure extended to both the internal and external network, both over the WiFi and the cable connections.
+All WiFi connections were dropped and we could no longer connect to the network.
 
-Abstract
-========
+Up until the issue started we were preparing to go on sky with the Auxiliary Telescope.
+The TMA was experiencing power issues and would not be able to operate during the night for the soak and start tracker testing. 
+Alysha Shugart and Tiago Ribeiro were executing some dome movement tests with the remote help of Bruno Quint.
 
-This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
+At one point around 8:30 PM local time, Tiago Ribeiro went to update LOVE in the love02 node with the latest version of the logging system for testing during the night.
+The node was completely shutdown (no LOVE system or ospl daemon running), shortly after the ospl daemon on love2 was initialized the network outage started.
+We were able to communicate with La Serena over Slack using a poor cellular connection and after being able to raise IT support it was established that it was impossible to recover from the issue remotely.
+
+At that point the crew decided to call in the night.
+The Auxiliary Telescope dome was manually closed and the telescope was left in the position it was in.
+The mirror covers were closed and all equipment was left in a safe state.
+
+The next morning IT arrived at the summit and determined that the issue was caused because two hosts were broadcasting using the same IP, which created an internal loop, causing the ACI to completely shutdown the network.
+They were able to determined that the IP responsible for the incident was 139.229.170.30, from the main ospl daemon, running on azar1, and the ospl daemon that was started on love2, initiating the episode.
+
+Mitigation Strategies
+=====================
+
+In summary, the key factors behind this incident are:
+
+* CISCO ACI network.
+* Brittle manual configuration required by DDS + docker
+
+In terms of mitigation strategies, it is worth noting that the CISCO ACI is already scheduled to be deprecated in early 2023.
+
+It is also worth noting that we are planning to retire DDS in favor of Kafka in the first quarter of 2023.
+Kafka does not use multicast traffic and requires a much simpler setup, with no need to setup daemons.
 
-Add content here
-================
+Altogether, the two main risk factors involved in this incident are already planned to be deprecated in a short time frame.
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Nevertheless, additional mitigation could be performed in case the Kafka adoption is delayed.
```

Comment:
"addition mitigation" -> "an addition mitigation" - since we only have one mitigation here.